src/routes/index.svelte

```diff
@@ -27,18 +27,21 @@
 </script>
 
 <script lang="ts">
-	import GameListHighlighted from '$lib/GameList/GameListHighlighted.svelte';
-	import GameListCarousel from '$lib/GameList/GameListCarousel.svelte';
+	/* Typings */
 	import type { Game } from '$lib/games';
 
+	/* Components */
+	import GameCardHighlighted from '$lib/Cards/GameCardHighlighted.svelte';
+	import Carousel from '$lib/GameViews/Carousel.svelte';
+
 	export let games: Game[];
 	const newestGames = games.slice(Math.max(games.length - 4, 0));
 </script>
 
-<main>
-	<GameListHighlighted />
-	<GameListCarousel gamesData={newestGames} carouselTitle="Nieuwe spellen" />
-	<GameListCarousel gamesData={games} carouselTitle="Favoriete spellen" />
+<main class="leaves-bg">
```

Comment:
Ik denk nog steeds dat we deze vaker in de app moeten laten terug komen dan alleen het speloverzicht

Reply:
Niks gezegd, wordt al gedaan 👍